recipes/dosfstools/dosfstools.inc

```diff
@@ -18,13 +18,13 @@ CFLAGS += '-D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64'
 
 PACKAGES =+ "${PN}-mkfs ${PN}-fsck ${PN}-fslabel"
 
-FILES_${PN}-mkfs = "${sbindir}/mkdosfs ${sbindir}/mkfs.*"
-FILES_${PN}-fsck = "${sbindir}/dosfsck ${sbindir}/fsck.*"
-FILES_${PN}-fslabel = "${sbindir}/dosfslabel"
+FILES_${PN}-mkfs = "${sbindir}/mkfs.fat"
+FILES_${PN}-fsck = "${sbindir}/fsck.fat"
+FILES_${PN}-fslabel = "${sbindir}/fatlabel"
 
-PROVIDES_${PN}-mkfs = "util/mkfs.msdos util/mkfs.vfat util/mkdosfs"
-PROVIDES_${PN}-fsck = "util/fsck.msdos util/fsck.vfat util/dosfsck"
-PROVIDES_${PN}-fslabel = "util/dosfslabel"
```

Comment:
Please take a look at the branch https://github.com/Villemoes/oe-lite-base/commits/ravi/dosfstools which I apparently failed to get any further with. I ran into a legacy script that called mkfs.vfat and was confused by the "lying" PROVIDE above until I found the --enable-compat-symlinks option. I don't think these legacy symlinks can hurt, and I don't even think they're worth a USE flag, but I'll leave it up to you whether to include them.